timeless/templates/auth/forgot_password_post.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block header %}
+  <h1>{% block title %}Forgot password{% endblock %}</h1>
+{% endblock %}
+{% block content %}
+  <h1>Forgot your password?</h1>
+  <p>We've sent an e-mail to {{email}} with your new password.</p>
+  <p>Please use it to log in and change it.</p>
+{% endblock %}
```

Comment:
@paulodamaso error is not used here!?

Reply:
@ammaratef45 When we get here we did not have any error, I've swapped the if and it is right now.